pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -261,6 +261,12 @@ func desiredLoadBalancerService(ci *operatorv1.IngressController, deploymentRef
 				}
 			}
 		}
+		// externalTrafficPolicy: Local is unsupported by OVN in
+		// OpenShift 4.9 and earlier; see
+		// <https://bugzilla.redhat.com/show_bug.cgi?id=2060542>.
+		if networkType == "OVN-Kubernetes" {
```

Comment:
its `OVNKubernetes`, we should probably hardcode tha value as a constant to avoid the typo?
reference: https://github.com/openshift/api/blob/fb6f933bb8d5ce8454a8777c0c4782c193ef5674/operator/v1/types_network.go#L536

Reply:
Thanks!  I didn't think to check for named constants because the `Spec.NetworkType` field of the `Network` type in `config/v1/types_network.go` is a string type.  I wonder whether the field could be changed to use the `NetworkType` type; I suppose if the same values are allowed, it may be all right to change the type, although it would require changes in any Go code using the Go type definitions.